app/models/scholarship.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# Clase de  relacion m a n entre grupos y users
```

Comment:
Consider updating the comments or removing them

Reply:
Done.